pandas/tests/test_nanops.py

```diff
@@ -1005,6 +1005,52 @@ def prng(self):
         return np.random.RandomState(1234)
 
 
+def _compare_nan_method_output(data, method, nan_method):
+    tm.assert_almost_equal(data.agg(method),
+                           data.agg(nan_method),
+                           check_exact=True)
+
+
+@pytest.fixture(params=[
+    pd.Series([1, 2, 3, 4, 5, 6]),
+    pd.DataFrame([[1, 2, 3], [4, 5, 6]])
+])
+def nan_test_object(request):
+    return request.param
+
+
+@pytest.mark.parametrize("standard, nan_method", [
+    (np.sum, np.nansum),
+    (np.mean, np.nanmean),
+    (np.std, np.nanstd),
+    (np.var, np.nanvar),
+    (np.median, np.nanmedian),
+    (np.max, np.nanmax),
+    (np.min, np.nanmin)
+])
+def test_np_nan_functions(standard, nan_method, nan_test_object):
+    _compare_nan_method_output(nan_test_object, standard, nan_method)
+    _compare_nan_method_output(nan_test_object, standard, nan_method)
```

Comment:
Are these two lines identical?

I think it'd be clearly to just write out the `tm.assert_almost_equal(nan_test_object.agg(standard), nan_test_object.agg(nan_method))`

Reply:
Yep, this was me being silly, fixed!